src/azure-cli/azure/cli/command_modules/sql/custom.py

```diff
@@ -3690,6 +3702,39 @@ def firewall_rule_create(
                                 end_ip_address=end_ip_address))
 
 
+#########################################################
+#           sql server outbound-firewall-rule           #
+#########################################################
+
+
+def outbound_firewall_rule_create(
+        client,
+        server_name,
+        resource_group_name,
+        outbound_rule_fqdn):
+    '''
+    Creates a new outbound firewall rule.
+    '''
+    return client.create_or_update(
+        server_name=server_name,
+        resource_group_name=resource_group_name,
+        outbound_rule_fqdn=outbound_rule_fqdn)
+
+
+def outbound_firewall_rule_delete(
+        client,
+        server_name,
+        resource_group_name,
+        outbound_rule_fqdn):
+    '''
+    Deletes an existing outbound firewall rule.
+    '''
+    return client.delete(
+        server_name=server_name,
+        resource_group_name=resource_group_name,
+        outbound_rule_fqdn=outbound_rule_fqdn)
```

Comment:
Can we delete this func since it's never used?

Reply:
Deleted.